pkg/gui/containers_panel.go

```diff
@@ -277,7 +277,17 @@ func (gui *Gui) refreshContainersAndServices() error {
 	gui.g.Update(func(g *gocui.Gui) error {
 		containersView.Clear()
 		isFocused := gui.g.CurrentView().Name() == "containers"
-		list, err := utils.RenderList(gui.DockerCommand.DisplayContainers, utils.IsFocused(isFocused))
+		toRender := gui.DockerCommand.DisplayContainers
```

Comment:
filtering the containers here won't work because we use the currently selected line index as the index on DisplayContainers so if you're hiding a heap of exited containers and now the first container in the list actually has index 10 in DisplayContainers, you won't see it's logs/graph. I've just tested to verify this.

We should instead do this filtering in gui.RefreshContainersAndServices() when we create DisplayContainers.